launch/tier4_planning_launch/config/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/obstacle_avoidance_planner.param.yaml

```diff
@@ -123,6 +123,8 @@
           eps_rel: 1.0e-10 # eps rel when solving OSQP
 
       mpt:
+        bounds_search_widths: [0.45, 0.15, 0.05, 0.01]
```

Comment:
Can you describe the meaning of the parameters in the readme?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/807/commits/73c6a9d6c0570bc96315aa6fe0947b417a9dd6c9
DONE